buildpipeline/DotNet-CoreFx-Trusted-Linux.json

```diff
@@ -402,6 +407,9 @@
     "PB_VsoRepositoryName": {
       "value": "DotNet-CoreFX-Trusted"
     },
+    "PB_VsoDockerFileDirUrl": {
+      "value": "https://$(PB_VsoAccountName):$(VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/DotNet-Buildtools-Trusted/src/Microsoft.DotNet.Build.Tasks/PackageFiles/scripts/docker/"
```

Comment:
While I'm OK with this as temp fix we need a more long term fix that doesn't depend on a unversioned repo endpoint. Can you please file a core-eng issue on this?